Gruntfile.js

```diff
@@ -181,7 +181,8 @@ module.exports = function(grunt) {
     */
   grunt.registerTask('nose', ['shell:nose-chrome', 'shell:nose-ie10']);
   grunt.registerTask('test', ['eslint', 'mocha_istanbul', 'nose']);
-  grunt.registerTask('build', ['squish', 'test']);
+  grunt.registerTask('test-js', ['eslint', 'mocha_istanbul']);
```

Comment:
The nose tests that are in this repo are not set up by default and are very time consuming to run, so I think it's best not to include them initial the build. Any suggestions for circumventing those in frontendbuild.sh?